src/libmongoc/tests/test-libmongoc.c

```diff
@@ -206,7 +206,7 @@ log_handler (mongoc_log_level_t log_level,
 
    suite = (TestSuite *) user_data;
 
-   if (log_level < MONGOC_LOG_LEVEL_INFO) {
+   if (log_level <= MONGOC_LOG_LEVEL_INFO) {
```

Comment:
I'm not sure why `INFO` messages were previously ignored, but this was necessary for the tests. Had a hell of a time figuring out why `ASSERT_CAPTURED_LOG` was failing before I found this.

Let me know if it should be called out in the commit message.